src/data/queries/healthCareLocalFacility.ts

```diff
@@ -194,60 +194,50 @@ export const formatter: QueryFormatter<
           fieldAlsoKnownAs: serviceTaxonomy?.field_also_known_as ?? '',
           fieldCommonlyTreatedCondition:
             serviceTaxonomy?.field_commonly_treated_condition ?? '',
+          fieldReferralRequired: healthService.field_referral_required,
           fieldTricareDescription:
             serviceTaxonomy?.field_tricare_description ?? null,
           // If it's TRICARE, use the TRICARE description. Otherwise, use the
           // regular description.
           description:
             healthService.field_administration.name === 'Lovell - TRICARE' && // Is this the best way to check?
             serviceTaxonomy?.field_tricare_description
-              ? serviceTaxonomy?.field_tricare_description // Should this also transform phone numbers into <va-telephone>s?
+              ? serviceTaxonomy?.field_tricare_description
```

Comment:
¿porqué?

I realize you just removed the comment; but, what is this line actually doing?

Reply:
We want to use the `description` field basically all time. Except for one page: Lovell TRICARE. For that one, we want to use the `field_tricare_description`. 🫠